specification/messaging/kafka.md

```diff
@@ -1,23 +1,94 @@
-<!--- Hugo front matter used to generate the website version of this page:
```

Comment:
I'm not sure what this do, but are we sure we can remove? Maybe @chalin can help us

Reply:
Please leave the Hugo front matter in.

Reply:
reverted